### PR TITLE
Merge branch 'master' into 0.next 

### PR DESCRIPTION
**Pull request by MasterOdin**
Necessary to merge #1681 against 0.next without having it include all of master's commits in the PR history and changeset.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ services:
 
 php:
   - 7.2
+  - 7.3
```
Comment (dereuromark) [inline .travis.yml]: the min/max should suffice here

File: docs/config/all.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#
+# Phinx documentation build configuration file, created by
+# sphinx-quickstart on Thu Jun 14 17:39:42 2012.
+#
+
+# Import the base theme configuration
+from cakephpsphinx.config.all import *
+
+# The full version, including alpha/beta/rc tags.
+release = '0.11.x'
```
Comment (dereuromark) [inline docs/config/all.py]: these might have to be fixed to 0.12 then

File: docs/en/install.rst [hunk 3]
```diff
@@ -10,7 +10,7 @@ website for more information.
 
 .. note::
 
-    Phinx requires at least PHP 5.4 (or later).
+    Phinx requires at least PHP 5.6 (or later).
```
Comment (dereuromark) [inline docs/en/install.rst]: 7.2 now

File: .travis.yml (hunk 1 above)
Comment (MasterOdin) [reply .travis.yml]: I'm not sure I follow this? Travis uses this as the list of versions to test, not a range, and so removing 7.3 would just cause Travis to not test 7.3 (see https://travis-ci.org/MasterOdin/phinx/builds/628789340 as example).

File: docs/en/install.rst (hunk 3 above)
Comment (MasterOdin) [reply docs/en/install.rst]: Done

File: docs/config/all.py (hunk 2 above)
Comment (MasterOdin) [reply docs/config/all.py]: Do you want me to change this or is this something that the cakephp folks will handle once 0.next is going to be merged into master?

File: .travis.yml (hunk 1 above)
Comment (dereuromark) [reply .travis.yml]: Yes, but there is no use case where min/max alone wouldnt suffice.

File: docs/config/all.py (hunk 2 above)
Comment (dereuromark) [reply docs/config/all.py]: If you can would be nice, so we dont forget when merging

File: composer.json [hunk 4]
```diff
@@ -27,9 +27,7 @@
     "require": {
         "php": ">=7.2",
         "cakephp/collection": "^4.0",
-        "cakephp/core": "^4.0",
         "cakephp/database": "^4.0",
-        "cakephp/datasource": "^4.0",
```
Comment (saeideng) [inline composer.json]: why these removed ?

Comment (MasterOdin) [reply composer.json]: Neither are directly depended upon and are just dependencies of dependencies.

Comment (saeideng) [reply composer.json]: fair enough 
yes , they don't use directly in phinx , so good to remove

File: docs/config/all.py (hunk 2 above)
Comment (MasterOdin) [reply docs/config/all.py]: Updated to 0.12.x

File: .travis.yml (hunk 1 above)
Comment (MasterOdin) [reply .travis.yml]: Removed 7.3

File: docs/config/all.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+# The marketing display name for the book.
+version_name = ''
+
+# Project name shown in the black header bar
+project = 'Phinx'
+
+# Other versions that display in the version picker menu.
+version_list = [
+    {'name': '0.11', 'number': '/phinx/11', 'title': '0.11', 'current': True},
```
Comment (dereuromark) [inline docs/config/all.py]: maybe we can make this version number more DRY in the future.

File: docs/config/all.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+# Languages available.
+languages = ['en', 'es', 'fr', 'ja']
+
+# The GitHub branch name for this version of the docs
+# for edit links to point at.
+branch = 'master'
+
+# Current version being built
+version = '0.11'
```
Comment (dereuromark) [inline docs/config/all.py]: here is another one

**Pull request merged**